pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -715,6 +716,14 @@ def test_to_datetime_unprocessable_input(self):
         )
         self.assertRaises(TypeError, to_datetime, [1, '1'], errors='raise')
 
+    @pytest.mark.parametrize('arg', [int(1), float(1), range(5),
```

Comment:
alternatively. We can simply remove the inheritance from ``tm.TestCase`` from this class. It was prob just copy-pasted, and prob not needed. (this is what causes parametrize to not work). 

you would need to change most ``self.assert*`` methods to use ``tm.assert*`` methods.